libraries/nomad_job.rb

```diff
@@ -20,16 +21,40 @@ def initialize(url, job_name)
   end
 
   def running?
-    job = http_json("#{@url}/v1/job/#{@job_name}")
+    job = get_job
     job['Status'] == 'running'
   end
 
+  def job_status
+    job = get_job
+    print_allocation_logs(job['ID'])
+    job_status = job['Status']
+  end
+
   def to_s
     "Nomad job #{@job_name} at #{@url}"
   end
 
   private
 
+  def get_job
+    job = http_json("#{@url}/v1/job/#{@job_name}")
```

Comment:
```suggestion
    http_json("#{@url}/v1/job/#{@job_name}")
```

Same as before, this will return the output of the `http_json` call.